try {
    hello()
} catch(error) {
    console.error(error)
}
  
try {
    hello()
} catch(error) {
    console.log(`Este es el error: ${error}`)
}